resources/quality/mingda/PETG/mingda_0.3_PETG_low.inst.cfg

```
@@ -13,3 +13,6 @@ variant = 0.3mm Nozzle
 [values]
 speed_layer_0 = 15
 wall_thickness = =line_width*4
+
+raft_interface_thickness = =0.2
```

```suggestion
raft_interface_thickness = 0.2
```
A direct value doesn't need the = in front of it. the extra = is to indicate that the value of the setting should be treated as a function.

---

ok